fq_split_rg_instructions.md

```diff
@@ -0,0 +1,105 @@
+# Get code and install in python env
+
+```sh
+git clone https://github.com/kmhernan/gdc-fastq-splitter
+cd gdc-fastq-splitter
+```
+*if you don't have it yet*, get virtual env for python3
+
+`pip3 install virtualenv # depends on your set up`
+
+```sh
+virtualenv venv --python /usr/bin/python3 # again, wherever your python3 (should be version 3.5+) is installed
+./venv/bin/pip install .
+venv/bin/gdc-fastq-splitter -h # if successful, you should get help menu
+```
+
+# Fun time run time
+
```

Comment:
should note that when there are paired-end reads (2 fastqs) it will use 2 cores

Reply:
Added comment